### ISSUE TITLE
Produce worse result after the scale parameter changed

### ISSUE THREAD

**Issue by yzslab** [NVlabs/instant-ngp#762]
Hi,
I use instant-ngp to train and render some scenes, but the results are not satisfied.
After reading some other issues (e.g. [this one](https://github.com/NVlabs/instant-ngp/issues/61#issuecomment-1016879118)) and the [Scaling existing datasets](https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md#scaling-existing-datasets), I realized that I should place the whole interested object into the unit cube.
But after setting the scale parameter value manually, instant-ngp produce a worse result.

The [Truck scene](https://drive.google.com/file/d/0B-ePgl6HF260NEw3OGN4ckF0dnM/view?usp=sharing&resourcekey=0-uYzL1Ga_EW1Ck0o-msT7Sg) from [Tanks and Temples](https://www.tanksandtemples.org/) is one of the scene I tested.
I use 172-196 as test set, and the others as training set. [Here is the processed dataset.](https://drive.google.com/file/d/1pkMinJplm9DbSfEj18d583SNhUliyAw5/view?usp=sharing)

After training for 50,000 iterations, the result wasn't satisfied.
As the below image show, the whole Truck doesn't fall in the unit cube:
![Truck doesn't fully fall in the unit cube](https://s2.loli.net/2022/07/10/15KrHWQpqFEcLCT.png)

So I manually set the `scale` parameter to `0.16`, which make the Truck fully fall into the unit cube:
![Truck fully fall in the unit cube](https://s2.loli.net/2022/07/10/7arGoSIt4JgFxyC.jpg)

But after training for 50,000 iterations, not a better result was produced:
| G.T.  | default (some part of the Truck fall outside the unit cube) |  fully fall in the unit cube  |
| ------------- | ------------- | ------------- |
|  ![000173.jpg_ref.png](https://s2.loli.net/2022/07/10/J1sa8EhrnK5FL7x.png)  |  ![000173.jpg_out.png](https://s2.loli.net/2022/07/10/uVyUOWMf2X4dCvE.png)  |  ![000173.jpg_out.png](https://s2.loli.net/2022/07/10/ok3pUrNh7fK69WQ.png)  |
|  ![000193.jpg_ref.png](https://s2.loli.net/2022/07/10/rOChuAIS7ngvpN1.png)  |  ![000193.jpg_out.png](https://s2.loli.net/2022/07/10/RkxcrN6YfVaTumg.png)  |  ![000193.jpg_out.png](https://s2.loli.net/2022/07/10/x8XG2IowWATPjFe.png)  |

As the first row of the images in table shown, after setting scale value, the right part of the Truck missing, and the rendered images didn't become more clear.

It seems that set the `near_distance` parameter to `0` can fix the "Truck missing" problem, but `near_distance=0` won't make the result better than default parameters, and will produce more floating artifacts:
|  default  |  scale=0.16, near_distance=0.2 (default)  |  scale=0.16, near_distance=0  |
| ------------- | ------------- | ------------- |
|  ![000173.jpg_out.png](https://s2.loli.net/2022/07/10/uVyUOWMf2X4dCvE.png)  |  ![000173.jpg_out.png](https://s2.loli.net/2022/07/10/sIxFwcMGr97ZPEg.png)  |  ![000173.jpg_out.png](https://s2.loli.net/2022/07/10/NmainhHJwQo9BXK.png)  |

I also found that change the `scale` and `near_distance` value will cause nearby objects to fail to render:
|  default parameters  |  scaled, near_distance=0  |
| ------------- | ------------- |
|  ![default.png](https://s2.loli.net/2022/07/10/Ai3CRVOajGodSE1.png)  |  ![scaled.png](https://s2.loli.net/2022/07/10/U2tTL3VJlzeCRsB.png)  |

As you can see, the model and paper disappeared! Both of the images are renderd from the same view point.

So,
Do you know how to scale the scene correctly?
Is there any extra parameters I need to adjust so as to improve the quality of Truck scene?
And would you mind telling me how the near_distance parameter works and how to set it correctly?

Thank you very much!
